src/controllers/cgnOperatorSearchController.ts

```diff
@@ -82,6 +83,21 @@ export default class CgnOperatorSearchController {
       );
     });
 
+  /**
+   * Count CGN merchants/discounts
+   */
+  public readonly count = (
+    req: express.Request
+  ): Promise<
+    | IResponseErrorValidation
+    | IResponseErrorInternal
+    | IResponseErrorNotFound
```

Comment:
`404` error case is not specified in the openapi spec

Reply:
there's no 404 on this endpoint, but it is just on a shared spec.